### ISSUE TITLE
Please do not kill this project.

### ISSUE THREAD

**Issue by ErikThiart** [adobe/brackets#15329]
I don't want to use VS code, that sucks.

Brackets is amazing, please don't kill me.

![2021-03-03_9-34-38](https://user-images.githubusercontent.com/9626641/109770379-2bb61a00-7c04-11eb-9285-549af044b98d.jpg)

**Comment by felipechalreo** [comment]
That sucks. I like Brackets.

**Comment by uchoamaster** [comment]
I loved Brackets, really sucks ...

**Comment by rxhull** [comment]
It's because of Adobe's deal with Microsoft so they're just promoting Visual Studio Code now

**Comment by lance1572** [comment]
Same here! I love brackets!

**Comment by fakerybakery** [comment]
Me too, I love Brackets! Please don't end it! I tried VS code, Atom, Whisk, WebStorm, everything! NOTHING is as good as Brackets!!! Brackets has the **BEST** live preview!!!

**Comment by LorenzoLotti** [comment]
T-T

**Comment by methilanka** [comment]
VSCODE is shit compared to brackets - comparing shortcuts and easy to use for front end ui projects. brackets is the best code editor i found in my 7 years career. killing brackets is SIN by adobe

**Comment by yhancik** [comment]
If the code is properly written and well documented, it can't really be killed. The question is more whether we can, as a community, make the development and support of a Brackets fork sustainable. Do we have/know devs in the community who are ready to tackle the task, and are we ready to chip in to help them?

**Comment by marko-hologram** [comment]
I loved Brackets when I used it few years ago when I started learning frontend development. I fell in love with the live preview feature, it was just really great compared to anything else I tried to use before. I haven't used Brackets in a long time since I switched to VS Code, but it's still sad to see it get deprecated. 

I mean, it's not unexpected since it didn't get a new release in ~10 months. It's not easy to maintain such a large project especially now since VS Code is dominating so much and can do everything Brackets can and 100x more really.

> 
> Emmet doesn't work, no live preview...

Did you actually try VS Code or you are just talking randomly? Personally, I never had problem with Emmet in any file format I wanted, even inside React Javascript files, it just works. Also, there is live preview, but as a plugin of course. I think you might be overreacting just a tiny bit 😄

**Comment by fakerybakery** [comment]
@marko-hologram: Maybe it was just the way I set it up...

**Comment by SolomonLeon** [comment]
If anyone is willing to continue to maintain it, please be sure to know me, thanks a lot!

**Comment by locness3** [comment]
Can the Brackets name be reused or is it a trademark?

**Comment by flopezre** [comment]
Any fork?

**Comment by locness3** [comment]
Not yet, apprently.

**Comment by kimmerin** [comment]
> 
> 
> Not yet, apparently.

There are 7000 forks. What do you mean by "not yet"?

**Comment by rafalkrupinski** [comment]
There should be a person or an organised group of people willing to maintain it, new name should be selected

**Comment by Dogleader6** [comment]
it's sad that they're removing brackets, but at least I know that someone will be maintaining it as there are a lot of passionate developers using this tool

**Comment by Alonso-del-Arte** [comment]
I was still using Adobe Photoshop Elements 2.0 when they first came out with Adobe CS. Not an exact analogy, but I can see myself using Adobe Brackets 1.14 past September. I have Visual Studio Code on this computer I'm using right now, but I almost never use that program. Then again, I don't really need Brackets for JavaScript, I use it almost entirely for HTML and sometimes CSS.

If you only want to use software that's currently supported, there are alternatives to Visual Studio Code, like Lite and Notepad++. I'm told Lite has all the features of Visual Studio Code but using much fewer system resources.

Regardless, this decision to discontinue Brackets is not a technological issue.

**Comment by webvars** [comment]
> Brackets has the **BEST** live preview!!!

The **Live Server** extension on vs code is better than Brackets. Even it can perform a live preview on Safari, Firefox, and Edge except Chrome. Try it then you might change your mind.

**Comment by yhancik** [comment]
I think we're having two parallel conversations here, and maybe we should split them into two different issues.

1. There's the question of whether there are people in the community ready to start & maintain a proper fork of Brackets
2. There's the question of alternatives, now that the future of Brackets is uncertain

I think that way we can have two constructive discussions, without turning it into a debate on whether it's still worth using (or wanting to use) Brackets ;)

**Comment by qaron84** [comment]
> I think we're having two parallel conversations here, and maybe we should split them into two different issues.
> 
> 1. There's the question of whether there are people in the community ready to start & maintain a proper fork of Brackets
> 2. There's the question of alternatives, now that the future of Brackets is uncertain
> 
> I think that way we can have two constructive discussions, without turning it into a debate on whether it's still worth using (or wanting to use) Brackets ;)

yes but in php sites live preview is horrid experience...

**Comment by Alonso-del-Arte** [comment]
> yes but in php sites live preview is horrid experience...

Now that sounds like a legitimate issue appropriate for Git. Have you or anyone else filed it?

**Comment by yhancik** [comment]
> I think that way we can have two constructive discussions, without turning it into a debate on whether it's still worth using (or wanting to use) Brackets ;)

Started two new issues on each of these topics : #15332 & #15333

**Comment by narayani28** [comment]
> 
> 
> Can the Brackets name be reused or is it a trademark?

Yes, the name Brackets can be reused, since the name is not an Adobe registered trademark.


Regards,
Narayani
Engineering Manager, Brackets 
Adobe

**Comment by scratchieguy12345678** [comment]
> If anyone is willing to continue to maintain it, please let me know, thanks a lot!

I'll might maintain it.

**Comment by yogo1212** [comment]
Hi :-)

Let's organise!

https://github.com/brackets-cont/brackets
